apps/extension/src/shared/components/display-origin-url.tsx

```diff
@@ -5,13 +5,13 @@ export const DisplayOriginURL = ({ url: { protocol, hostname, port } }: { url: U
       <span className='tracking-[-0.21em]'>{':/'}</span>
       {'/'}
     </span>
-    <span className='tracking-[-0.05em] brightness-150 saturate-150'>{hostname}</span>
+    <span className='-tracking-wider brightness-150 saturate-150'>{hostname}</span>
```

Comment:
can we use `tracking-tighter` instead of `-tracking-wider`?

Reply:
ah true! Will fix.